<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcustomerdialog</class>
 <widget class="QDialog" name="editcustomerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Customer</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>New Last Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>New Phone</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="editCustomerDialogNewPhone"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>New First Name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="editCustomerDialogNewLastName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Customer ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="editCustomerDialogNewFirstName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="editCustomerDialogSelectCustomerID"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>431</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="editCustomerDialogSaveButton">
      <property name="styleSheet">
       <string notr="true">background: rgb(78, 154, 6)</string>
      </property>
      <property name="text">
       <string>Save Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editCustomerDialogCancelButton">
      <property name="styleSheet">
       <string notr="true">background: rgb(164, 0, 0)</string>
      </property>
      <property name="text">
       <string>Cancel Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="editCustomerDialogConfirmationMessage">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>370</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
